modules/users/users.controller.js

```diff
@@ -32,16 +35,39 @@ const createUser = async (req, res) => {
       email: req.body.email,
       subscription: "starter",
       avatarURL: gravatar.url(req.body.email),
+      verificationToken: uuidv4(),
     });
     user.setPassword(req.body.password);
 
     await user.save();
+
+    sgMail.setApiKey(process.env.API_TOKEN);
```

Comment:
To wystarczy dodać tylko raz przy inicjalizacji, a nie za każdym razem jak wykonujemy funkcję